tests/Test/AbstractFixerTestCase.php

```diff
@@ -166,7 +166,7 @@ final public function testFixerDefinitions(): void
             $duplicatedCodeSample = array_search(
                 $sample,
                 \array_slice($samples, 0, $sampleCounter),
-                false
+                true
```

Comment:
why?

Reply:
I'd rather as why it was `false` in the first place. This is the error from `phpstan-strict-rules`:

> Call to function array_search() requires parameter #3 to be set.

To be honest it does not matter - samples are always string, so strict or not strict will work the same.